rllib/evaluation/rollout_worker.py

```diff
@@ -492,8 +494,19 @@ def gen_rollouts():
 
         if self.env is not None:
             # Validate environment (general validation function).
-            _validate_env(self.env, env_context=self.env_context)
-            # Custom validation function given.
+            if not self._disable_env_checking:
+                logger.warning(
+                    "We've added a module for checking environments that "
+                    "are used in experiments. It will cause your "
+                    "environment to fail if your environment is not set up"
+                    "correctly. You can disable check env by setting "
+                    "`disable_env_checking` to True in your experiment config "
+                    "dictionary. You can run the environment checking module "
+                    "standalone by calling ray.rllib.utils.check_env(env)."
+                )
+                check_env(self.env)
```

Comment:
can you:

```
try:
    check_env(self.env)
except:
    logger.waring(...)
    raise
```
?
no reason to scare the user if everything is good right?

Reply:
but if check_env fails, then users should know that their environment failed environment checking, and rectify the situation immediately by either

Reply:
ok, what I meant was: why do we want to print logger.warning() if their env is fine and passes the check?

Reply:
ah I understand.

Reply:
yeah lemme implement that logic instead.